lib/_tls_common.js

```diff
@@ -100,15 +100,12 @@ exports.createSecureContext = function createSecureContext(options) {
 
   const c = new SecureContext(options.secureProtocol, secureOptions,
                               options.minVersion, options.maxVersion);
-  let i;
-  let val;
 
   // Add CA before the cert to be able to load cert's issuer in C++ code.
   const { ca } = options;
   if (ca) {
     if (ArrayIsArray(ca)) {
-      for (i = 0; i < ca.length; ++i) {
-        val = ca[i];
+      for (const val in ca) {
```

Comment:
Shouldn’t this be `of` instead of `in`? (I’d definitely prefer the indexed variant over `for ... in` loops.)

Reply:
My bad, I'm making these changes in auto mode while attending CollabSummit and letting CI do the checking.

Thanks for pointing out - will fix!

Reply:
Resolved in https://github.com/nodejs/node/pull/30961/commits/c0cd5b21c87e0e0969d39dc37dcd33233959bca8

Thank you @addaleax for pointing it out!